release/models/platform/openconfig-platform-transceiver.yang

```diff
@@ -843,6 +849,22 @@ module openconfig-platform-transceiver {
       }
       units volts;
     }
+    leaf module-temperature-lower {
```

Comment:
is there any significance to "module" here?  Can this just be `temperature-low` and `temperature-high`?

Reply:
I used module to differentiate from `laser-temperature-*`